### PR TITLE
Refactor index.jade to be dynamic

### PR DESCRIPTION
**Pull request by raphaella-rose**

### CODE REVIEW

File: cypress/e2e/bookshop_details_spec.cy.js [hunk 1]
```diff
@@ -1,56 +1,49 @@
...
-    cy.get(".bookshop-address").last().should("contain", "SW9 8PS");
+    cy.get("#gaystheword").find(".bookshop-address").should("contain", "66 Marchmont Street");
+    cy.get("#gaystheword").find(".bookshop-address").should("contain", "London");
+    cy.get("#gaystheword").find(".bookshop-address").should("contain", "WC1N 1AB");
+
+    cy.get("#roundtablebooks").find(".bookshop-address").should("contain", "97 Granville Arcade");
+    cy.get("#roundtablebooks").find(".bookshop-address").should("contain", "Coldharbour Lane");
+    cy.get("#roundtablebooks").find(".bookshop-address").should("contain", "Brixton");
+    cy.get("#roundtablebooks").find(".bookshop-address").should("contain", "London");
+    cy.get("#roundtablebooks").find(".bookshop-address").should("contain", "SW9 8PS");
```
Comment (kaykakaraka) [inline cypress/e2e/bookshop_details_spec.cy.js:19]: great! yes this solves future problems

File: cypress/e2e/homepage_spec.cy.js [hunk 2]
```diff
@@ -13,11 +13,13 @@ describe('The homepage', () => {
...
         cy.visit('/openbook');
         cy.get('nav').should('be.visible');
+    
+    })
 
     it("should redirect to homepage when url is '/' ", () => {
         cy.visit('/');
 
         cy.url().should('eq', 'http://localhost:3030/openbook')
     })
```
Comment (kaykakaraka) [inline cypress/e2e/homepage_spec.cy.js:22]: nice!

**Review by BenjaminNeustadt — APPROVED** [review]

**Review by edra-dev — APPROVED** [review]

File: index.js [hunk 3]
```diff
@@ -33,8 +33,14 @@ app.get("/", (req, res) => {
...
   Bookshop.find((err, bookshops) => {
-    res.render('index', { bookshops: bookshops } )
+    bookshops.forEach((bookshop) => {
+      let shop = bookshop.name.replace(/'/g, "").split(" ")
+      shop = shop.join("").toLowerCase()
+      bookshopIds.push(shop)
+    })
+    res.render('index', { bookshops: bookshops, bookshopIds: bookshopIds })
   })
 });
```
Comment (kaykakaraka) [inline index.js:46]: my understanding is that ideally we will push this into the model but we could do this in a refactor

File: frontend/views/index.jade [hunk 4]
```diff
@@ -16,37 +16,22 @@ block content
...
+          div(class='opening-hours') Opening hours
+          div(class='times')
+            each item in bookshops[i].openingHours
+              | #{item}
+              br
+          div(class='tags') 
+            each tag in bookshops[i].tags
+              | ##{tag} 
+     - }
+
```
Comment (kaykakaraka) [inline frontend/views/index.jade:37]: nice

**Review by kaykakaraka — APPROVED** [review]
I really like this pull request

**Pull request merged**